formula.css

```diff
@@ -1,11 +1,10 @@
 html {
   font-family: monospace;
-  font-size: calc(1ex + 2vh + 3vw);
+  font-size: calc(1ch + 2vmin + 3vmax);
```

Comment:
the base `ch` appears a hair wider than `ex` while still keeping simple numbers. the scaling change to `vmin` `vmax` grows portrait more than before and landscape as it was because `vmin` and `vmax` switch between orientation modes.

Reply:
font scaling recipes fyi @jrdnrch